synth.metadata

```diff
@@ -1,19 +1,19 @@
 {
```

Comment:
why your PR contains this file?

Reply:
it was from merging from master, i cleaned it.